src_js/grndwork_api_client/client.ts

```diff
@@ -19,41 +22,87 @@ export class Client {
   ) {}
 
   public getStations(
-    query: GetStationsQuery = {},
-    pageSize = 100,
+    query: GetStationsQuery | null = null,
+    pageSize: number | null = null,
   ): IterableResponse<Station> {
-    const {refreshToken, platform} = this;
+    const iterator = this._requestStations(query, pageSize);
 
-    async function* request(): AsyncIterableIterator<Station> {
-      const accessToken = await getAccessToken(refreshToken, platform, 'read:stations');
+    return new IterableResponse(iterator);
+  }
 
-      yield* makePaginatedRequest<Station>({
-        url: STATIONS_URL,
-        token: accessToken,
-        query,
-      }, pageSize);
-    }
+  private async* _requestStations(
+    query: GetStationsQuery | null,
+    pageSize: number | null,
+  ): AsyncIterableIterator<Station> {
+    const {refreshToken, platform} = this;
+    const accessToken = await getAccessToken(refreshToken, platform, 'read:stations');
 
-    return new IterableResponse(request());
+    yield* makePaginatedRequest<Station>({
+      url: STATIONS_URL,
+      token: accessToken,
+      query: query || {},
+    }, pageSize || 100);
   }
 
   public getData(
-    query: GetDataQuery = {},
-    pageSize = 100,
+    query: GetDataQuery | null = null,
+    includeQCFlags: boolean | null = null,
+    pageSize: number | null = null,
   ): IterableResponse<DataFile> {
+    let iterator = this._requestDataFiles(query, pageSize);
+
+    if ((query || {}).records_limit && includeQCFlags !== false) {
+      iterator = this._includeQCFlags(iterator);
+    }
+
+    return new IterableResponse(iterator);
+  }
+
+  private async* _requestDataFiles(
```

Comment:
Why the `_` on the method name. It is marked as private. Is this to match the names in python?